pom.xml

```diff
@@ -98,6 +98,13 @@
         <version>${version.org.jgrapht}</version>
       </dependency>
 
+      <!-- servlet dependencies -->
+      <dependency>
+        <groupId>org.jboss.spec.javax.servlet</groupId>
+        <artifactId>jboss-servlet-api_3.0_spec</artifactId>
```

Comment:
Looking more, I should probably change the scope to `provided` as in the end the specification available in the EAP/WildFly shall be used.